guzzlehttp/guzzle/CVE-2016-5385.yaml

```diff
@@ -0,0 +1,8 @@
+title:     HTTP Proxy header vulnerability
+link:      https://github.com/guzzle/guzzle/releases/tag/6.2.1
+cve:       CVE-2016-5385
+branches:
+    master:
+        time:     2015-07-15 17:14:23
+        versions: ['>4.0.0rc2 <6.2.1']
```

Comment:
Actually, it is fixed respectively in version 4.2.4, 5.3.1, and 6.2.1 for the 4.x, 5.3, and master branches.